.rubocop.yml

```diff
@@ -23,6 +23,7 @@ Metrics/MethodLength:
 Metrics/PerceivedComplexity:
   Exclude:
     - "lib/google/gax/config.rb"
+    - "lib/google/gax/grpc/stub.rb"
```

Comment:
I'm going to submit an issue to clean up these exclusions. Ideally we won't be excluding "normal" files from our rubocop settings.